mithril-common/src/chain_observer/pallas_observer.rs

```diff
@@ -234,8 +234,29 @@ impl PallasChainObserver {
         Ok(Some(stake_distribution))
     }
 
-    /// Calculates the current KES period
-    /// using the provided `chain_point`and `slots_per_kes_period`.
+    /// # Calculate Current KES Period
+    ///
+    /// It calculates the current Key Evolving Signature (KES) period
+    /// based on the provided `chain_point` and `slots_per_kes_period`.
+    ///
+    /// The calculation formula is represented as:
+    ///
+    /// `floor(P / S)`
+    ///
+    /// where:
+    /// - `P` represents the current point on the chain (slot number).
+    /// - `S` symbolizes the number of slots in each KES period.
+    /// - `floor(.)` is the floor function which rounds down a number to the nearest whole.
```

Comment:
```suggestion
    /// `current_kes_period = ⌊current_slot_number/slots_per_kes_period⌋`
    ///
    /// where:
    /// - `current_slot_number` represents the current slot number given by the `point` on the chain.
    /// - `slots_per_kes_period` represents the number of slots in a KES period.
    /// - `⌊x⌋` is the floor function which rounds the greatest integer less than or equal to `x`.
```

Reply:
done

Reply:
GitHub didn't updated my commits, it seems to be a sync problem.

the pushes doesn't go so let's wait and see when the new commits arrives 👀

Reply:
![CleanShot 2024-03-01 at 15 55 07@2x](https://github.com/input-output-hk/mithril/assets/33763843/770dd4e4-9d15-4659-9e8a-636d357458c7)

Reply:
Yes, there is an incident with PRs and actions: https://www.githubstatus.com/incidents/wcl1sw4mzg60

Reply:
done